shipcat_definitions/src/structs/authorization.rs

```diff
@@ -6,14 +6,14 @@ pub struct Authorization {
 
     /// Are anonymous requests allowed to reach the service?
     ///
-    /// If true, requests with no `Authorization` header (or an invalid/expired JWT) will be proxied to the service (but will receive an `Anonymous: true` header)
+    /// If true, requests with no `Authorization` header (or an invalid/expired JWT, depending on reject_invalid_tokens) will be proxied to the service (but will receive an `Anonymous: true` header)
     /// If false, they will be rejected (with a 401 response)
     pub allow_anonymous: bool,
 
-    /// Should invalid/expired tokens be stripped from the upstream request?
+    /// Should requests with invalid/expired tokens be disallowed from reaching the service?
```

Comment:
```suggestion
    /// Are requests with invalid/expired tokens allowed to reach the service?
```

Same format as `allow_anonymous`